src/main/java/racingcar/Car.kt

```diff
@@ -0,0 +1,27 @@
+package racingcar
+
+data class Car(val name: String) {
+    var currentPosition = 0
+        private set
+
+    init {
+        if (name.length > 5) {
+            throw IllegalArgumentException()
+        }
+    }
```

Comment:
어떤 문제가 있는지 사용자에게 정확히 안내해주는 건 어떨까요? 😉 
아래와 같이 require 를 활용해볼 수도 있겠어요.

```suggestion
    init {
        require(name.length <= 5) { "이러쿵 저러쿵" }
    }
```